pandas/tests/arrays/test_boolean.py

```diff
@@ -546,12 +546,29 @@ def test_reductions_return_types(dropna, data, all_numeric_reductions):
 #         result = arr[mask]
 
 
-@pytest.mark.skip(reason="broken test")
 @td.skip_if_no("pyarrow", min_version="0.15.0")
 def test_arrow_array(data):
     # protocol added in 0.15.0
     import pyarrow as pa
 
     arr = pa.array(data)
-    expected = pa.array(np.array(data, dtype=object), type=pa.bool_(), from_pandas=True)
+
+    # TODO use to_numpy(na_value=None) here
```

Comment:
is this todo actionable now or do we have to wait until [condition]?

Reply:
It's not yet implemented, open issue for this is https://github.com/pandas-dev/pandas/issues/30038